### ISSUE TITLE
Break functions apart

### ISSUE THREAD

**Issue by VOSTPortugal** [vostpt/bot#1]
At this time the bot works in a fantastic way but:
- We have an unique code file for all the functions, resulting in having actionable data and _just for fun_ content in the same file. 

**Proposal**

- Clean the code
- Validate the code
- Comment the code (this is important so everyone knows what is going on with the code)
- Separate the code into multiple files, one for each use case.

**Comment by JorgeMiguelGomes** [comment]
Use this base code (already tested but can be optimised) to streamline main file

```JS
// VOST Portugal Bot  version 2.0 

// Requirements 

const fs = require('fs');
const Discord = require('discord.js');
const { prefix, token } = require('./config.json');
const fun = require('./fun');

// const axios = require('axios');
// const schedule = require('node-schedule');
// const moment = require('moment');
// moment.locale('pt');
// var Twitter = require('twitter');

// Defines bot client, command's cooldowns, commands, points to commands' directory
const client = new Discord.Client();
const cooldowns = new Discord.Collection();
client.commands = new Discord.Collection();
const commandFiles = fs.readdirSync('./commands').filter(file => file.endsWith('.js'));

// defines commands 
for (const file of commandFiles) {
	const command = require(`./commands/${file}`);
	client.commands.set(command.name, command);
}
// Opens client

client.on('message', message => {
    // if command is sent by bot returns back to listening
   if (!message.content.startsWith(prefix) || message.author.bot) return;

    const args = message.content.slice(prefix.length).split(/ +/);
	const commandName = args.shift().toLowerCase();

    const command = client.commands.get(commandName)
	|| client.commands.find(cmd => cmd.aliases && cmd.aliases.includes(commandName));
    // If command doesn't exist returns a message 
    if (!command) return message.channel.send(`Ainda não conheço esse comando ${message.author}`); 
    // If command's syntax lacks arguments returns message 
    if (command.args && !args.lenght) {
        return message.channel.send (`Não me deste os dados suficientes para trabalhar ${message.author} :thinking:`);
    }
    // Cooldowns Setup 
    if (!cooldowns.has(command.name)) {
        cooldowns.set(command.name, new Discord.Collection());
    }

    const now = Date.now();
    const timestamps = cooldowns.get(command.name);
    const cooldownAmount = (command.cooldown || 3) * 1000;
    
    if (timestamps.has(message.author.id)) {
        const expirationTime = timestamps.get(message.author.id) + cooldownAmount;
    
        if (now < expirationTime) {
            const timeLeft = (expirationTime - now) / 1000;
            return message.reply(`Aguenta aí os cavalos e espera ${timeLeft.toFixed(1)} segundos antes de usares o comando \`${command.name}\` outra vez`);
        }
    }
    timestamps.set(message.author.id, now); 
    setTimeout(() => timestamps.delete(message.author.id), cooldownAmount);

    // Executes command. If command returns error it sends a message to the user. 
    try {
        command.execute(message, args);
    } catch (error) {
        console.error(error);
        message.reply('Aconteceu um erro ao processar o comando :thinking: Já despedi o estagiário!');
    }

    
	
});

client.login(token);
```

**Comment by tiagojpdias** [comment]
Fixed at #12
